### PR TITLE
Load ToC and Privacy Policy through pdf documents

### LINKED ISSUE #121: Move membership terms and conditions to an external pdf file and render in app

### PR DESCRIPTION
**Pull request by ahmedhamodi**
close #121

### CODE REVIEW

**Review by arsidada — CHANGES_REQUESTED** [review]
@ahmedhamodi can you post screenshots for the before and after so we can visually review the change

File: src/Components/Registration/FinalPage.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ import Tooltip from "@material-ui/core/Tooltip";
 import Membership from "../LandingPage/Membership";
 import Stripe from "../Payment/Stripe";
 import Landing from "../LandingPage/Landing";
+import { Document, Page, pdfjs } from "react-pdf";
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
```
[the remark this answers is not in the document]
Comment (ahmedhamodi) [reply src/Components/Registration/FinalPage.js:25]: The PDF file fails to load if that cdn isnt included/working:
image

How do you recommend we work around this? The main solution I found online is to use this pdf worker.

**Review by mohaimenhasan — APPROVED** [review]
Looks good to me !!

**Comment by ahmedhamodi** [comment]
Can you re-review @arsidada?

Here are the before/after pictures you requested.

Before:
image

After:
image

**Review by arsidada — APPROVED** [review]

**Pull request merged**